guides/source/configuring.md

```diff
@@ -455,11 +455,11 @@ to `'http authentication'`.
 * `config.action_dispatch.signed_cookie_salt` sets the signed cookies salt value.
 Defaults to `'signed cookie'`.
 
-* `config.action_dispatch.encrypted_cookie_salt` sets the encrypted cookies salt
-value. Defaults to `'encrypted cookie'`.
+* `config.action_dispatch.encrypted_cookie_salt` sets the encrypted cookies salt value. This is value is deprecated in Rails 5.2 in favor of `config.action_dispatch.authenticated_encrypted_cookie_salt`.
 
-* `config.action_dispatch.encrypted_signed_cookie_salt` sets the signed
-encrypted cookies salt value. Defaults to `'signed encrypted cookie'`.
+* `config.action_dispatch.encrypted_signed_cookie_salt` sets the signed encrypted cookies salt value. This is value is deprecated in Rails 5.2 in favor of `config.action_dispatch.authenticated_encrypted_cookie_salt`.
```

Comment:
Just saying deprecated in the guides won't cut it. We need to use `ActiveSupport::Deprecation.warn` — but I'm not sure we should deprecate those just yet, so let's leave this mention out of this PR.